smslibrary/src/main/java/com/eis/communication/Header.java

```diff
@@ -0,0 +1,90 @@
+package com.eis.communication;
+
+/**
+ * Header abstract class
```

Comment:
Be more explicative when adding specifications on top of a class: what does this class represent? How should it be used?